### ISSUE TITLE
sgx-setup interferes with substrate node

### ISSUE THREAD

**Issue by haerdib** [integritee-network/sgx-setup#5]
Intel sgx sdk comes with it own ld, as, & other binutilites versions:

https://github.com/integritee-network/sgx-setup/blob/main/sgx-ansible/files/install-sgx-sdk.sh#L7

However, they do not work with the recent substrate versions: https://github.com/integritee-network/integritee-node/issues/97
 ( I have the strong suspision that the recently updated [wasmtime](https://github.com/paritytech/substrate/pull/10149/files) is the cause, due to new dependency on [linux-raw-sys](https://github.com/sunfishcode/linux-raw-sys), but I didn't investigate further there.)
 
 I have tried to simply remove the intel specifc binutil programms on Ubuntu 18.04 and both, worker and node, are compiling. 

So for now, simply adding binutils to the install-tools script and remove the copy part of install-sgx-sdk, seems to do the trick.

As reported by Basti on Element chat, version 2.35.2 also works with substrate. We might as well do an upgrade (apt does not provide 2.35.2, so we'd probably need to take it from: http://ftp.gnu.org/gnu/binutils/)

But whichever version we take, we should not take the one that intel provides.

**Comment by haerdib** [comment]
Actually - building the sdk itself is not necessary anymore. We can just take the prebuilt version provided by intel (https://download.01.org/intel-sgx/latest/linux-latest/distro/ubuntu20.04-server/) and install it. That will solve this problem as well.

**Comment by haerdib** [comment]
Tried that on branch https://github.com/integritee-network/sgx-setup/tree/use-prebuilt-intel-binaries and it works perfectly (never installed intel sgx so fast).

Only problem: apt always takes the newest versions by default, including dependencies, even when the original library was specified to take a specific version. That interferes heavily in case an older sdk version should be installed (like in the case of devsgx01). After searching for a nice apt solution I decided this is too error-prone (might work now, but in the future it might easily be broken). So I decided to stay on the old version - just build the sdk version and take the psw version from there (with apt).
